nmodl/ode.py

```diff
@@ -25,9 +26,12 @@
     raise ImportError(f"Requires SympPy version >= 1.2, found {major}.{minor}")
 
 def _get_custom_functions(fcts):
+    forbidden_var = ["beta", "gamma"]
```

Comment:
this could be somewhere at the top level. And also, there more functions than 2 IIRC. (Alessandro listed those).